broker/src/tunneldigger_broker/broker.py

```diff
@@ -82,6 +85,15 @@ def create_tunnel(self, broker, address, uuid, remote_tunnel_id, client_features
         if self.last_tunnel_created is not None and now - self.last_tunnel_created < self.connection_rate_limit:
             logger.info("Rejecting tunnel %s due to rate limiting" % tunnel_str)
             return False
+        if address[0] in self.last_tunnel_created_per_ip and now - self.last_tunnel_created_per_ip[address[0]] < self.connection_rate_limit_per_ip:
```

Comment:
So the est here is simply "2 attempts from the same IP in a short time"? isn't that a bit simplistic? It might be better to make the number of attempts configurable, like "when there are 5 attempts within 20 seconds". However, that would likely also be harder to implement (I am not sure what the usual implementation strategy for this is).

Reply:
Okay. I changed this method to an adjustable one.